index.html

```diff
@@ -285,6 +285,33 @@ <h1>Skills</h1>
         </div>
     </section> 
     
+    <section class="contact-form">
+        <h1 class="title-contact">Contact Me</h1>
+        <h2 class="view">If you have an application you are interested in developing,
+            a feature that you need built or a project that needs coding. 
+            I’d love to help with it 
+        </h2>   
+        
+        <form>
+            <input type="text" class="input" placeholder="Full name" maxlength="30">
+            <br>
+            <br>
+            <input type="email" class="input" placeholder="Email address">
+            <br>
+            <br>         
+            <input type="text" class="box" placeholder="Enter text here" maxlength="500">
+            <br>
+            <br>
+            <div class="get-in-touch">
+                <button type="button"> Get In Touch </button>
+            </div> 
+
+        </form>
```

Comment:
- [ ] Make sure placeholders are the same as in the original design. Of course, you will change your name and mail with your name and mail.
- [ ] All input fields should have the `required` attribute. Please add it.
- [ ] Form uses the action and method attribute to integrate with formspree service. Please fix it.
[Requirement.
](https://formspree.io/html)
![image](https://user-images.githubusercontent.com/38635158/149845722-35e578df-3be0-4944-a160-94648f5397eb.png)
- [ ] Message input's should be `<textarea>`. Please change it

Reply:
textarea added, required added.